src/components/crossForm/CrossForm.vue

```diff
@@ -553,11 +552,10 @@ export default {
       }
       const accountAddress = this.sharedState.accountAddress
       const tokenAddress = this.selectedToken.address
-      const maxValue = new BigNumber(parseInt(this.amount)).shiftedBy(this.selectedToken.decimals);
       
       try {
         this.showSpinner = true
-        const receipt = await this.erc20TokenInstance.approve(tokenAddress, maxValue.toString(), {
+        const receipt = await this.erc20TokenInstance.approve(tokenAddress, {
```

Comment:
First was changed to approve only the value that the user want to cross on amount field. 
After that I changed back to maximum integer so the token it's approved for life by the user.